variables.tf

```diff
@@ -125,15 +141,32 @@ variable "vcenter_user_name" {
 }
 
 variable "s3_url" {
+  default = "https://s3.example.com"
 }
 
 variable "s3_bucket_name" {
+  default = "vmware"
 }
 
 variable "s3_access_key" {
+  default = "S3_ACCESS_KEY"
 }
 
 variable "s3_secret_key" {
+  default = "S3_SECRET_KEY"
+}
+
+variable "s3_boolean" {
```

Comment:
Perhaps this could be renamed to fit how it is used.  Ultimately, this defines which tool will be used. I have encountered some object storage services (based on Rook) that don't work well with `mc`. We could more easily allow for extra tool choices if this field was not a boolean but a tool name: "mc" vs "gcs".

Reply:
I see your point. I could change this and collapse the variable for `s3_bucket` and `gcs_bucket` to just `object_store_bucket`

Reply:
Changed.